app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -0,0 +1,177 @@
+class MiqCockpitWsWorker::Runner < MiqWorker::Runner
+  BINDING_ADDRESS = ENV['BINDING_ADDRESS'] || (Rails.env.production? ? nil : "0.0.0.0")
```

Comment:
I'd prefer we be explicit with what we want in production:  talk only to localhost.